<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TariffAddDialog</class>
 <widget class="QDialog" name="TariffAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Добавление</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QFormLayout" name="formLayout">
               <property name="horizontalSpacing">
                <number>0</number>
               </property>
               <property name="verticalSpacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item row="0" column="0" colspan="2">
                <widget class="QGroupBox" name="groupBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="title">
                  <string>Основные параметры</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Дата начала</string>
                    </property>
                    <property name="buddy">
                     <cstring>edtDate</cstring>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDateEdit" name="edtDate"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Тариф</string>
                    </property>
                    <property name="buddy">
                     <cstring>edtTariff</cstring>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="edtTariff">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>Дополнительные параметры</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Тарифицируется</string>
                    </property>
                    <property name="buddy">
                     <cstring>cmbTariffType</cstring>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cmbTariffType">
                    <property name="currentIndex">
                     <number>2</number>
                    </property>
                    <item>
                     <property name="text">
                      <string>Посещение</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Событие</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Мероприятие по количеству</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Визит-день</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Событие по койко-дням</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Мероприятие по УЕТ</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Мероприятие по количеству и тарифу койки</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Визиты по мероприятию</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Визиты по МЭС</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Событие по МЭС</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Событие по МЭС с учётом длительности</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Визиты по профилю</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Событие по МЭС с учётом уровня выполнения и длительности</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Краснодар,П13: событие по КСГ с учётом длительности и операций</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Мурманск,2015: стационар</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Ед. Уч.</string>
                    </property>
                    <property name="buddy">
                     <cstring>cmbAid</cstring>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="CRBComboBox" name="cmbAid"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Фильтр</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QCheckBox" name="chkContract">
                 <property name="text">
                  <string>По договору</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="edtFilter"/>
               </item>
               <item>
                <widget class="QPushButton" name="btnApplyfilter">
                 <property name="text">
                  <string>Применить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="CInDocTableView" name="tblService"/>
            </item>
            <item>
             <widget class="QLabel" name="lblChecked">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnClear">
                 <property name="text">
                  <string>Очистить выбор</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnAll">
                 <property name="text">
                  <string>Выделить все</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_5" native="true">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnCancel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Отмена</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Добавить</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Замена</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>На дату:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="edtDateChange"/>
         </item>
         <item>
          <widget class="QCheckBox" name="chkOnlyUET">
           <property name="text">
            <string>Только тарифы с УЕТ</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QTableWidget" name="tblTariff">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Список услуг с выбранным тарифом</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="CTableView" name="tblTariffServices"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="btnCancelChange">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Отмена</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveChange">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Сохранить</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtDate</tabstop>
  <tabstop>edtTariff</tabstop>
  <tabstop>cmbTariffType</tabstop>
  <tabstop>cmbAid</tabstop>
  <tabstop>tblService</tabstop>
  <tabstop>btnClear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
